instrumentation/play/play-mvc/play-mvc-2.4/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/play/v2_4/server/PlayServerTest.java

```diff
@@ -112,9 +115,15 @@ protected void configure(HttpServerTestOptions options) {
   }
 
   @Override
+  @SuppressWarnings("deprecation") // uses deprecated semconv
   public SpanDataAssert assertHandlerSpan(
       SpanDataAssert span, String method, ServerEndpoint endpoint) {
-    span.hasName("play.request").hasKind(INTERNAL);
+    span.hasName("play.request")
+        .hasKind(INTERNAL)
+        .hasAttributesSatisfyingExactly(
+            equalTo(CODE_NAMESPACE, "play.api.mvc.ActionBuilder$$anon$2"),
+            equalTo(CODE_FUNCTION, "apply"));
```

Comment:
In my opinion this is not useful. It points to framework code and is the same for all endpoints.